test/packaging/ansible/agent-upgrade.yml

```diff
@@ -56,7 +56,7 @@
             name: agent-upgrade
           vars:
             # this used only for check that latest is grater than given
-            target_agent_version: "1.27.3"
+            target_agent_version: "1.27.2"
```

Comment:
as the comment says this is used to assert that the latest one is bigger that this one